src/main/scala/org/apache/mesos/chronos/scheduler/mesos/MesosJobFramework.scala

```diff
@@ -28,7 +28,8 @@ class MesosJobFramework @Inject()(
                                    val config: SchedulerConfiguration,
                                    val frameworkIdUtil: FrameworkIdUtil,
                                    val taskBuilder: MesosTaskBuilder,
-                                   val mesosOfferReviver: MesosOfferReviver)
+                                   val mesosOfferReviver: MesosOfferReviver,
+                                   val constraintChecker: ConstraintChecker = new ConstraintChecker())
```

Comment:
thanks gaston, i've got decades of java, but not so much scala, so i appreciate these tips. 

i'll try another pass asap. :+1: